models/article_id_comments.model.js

```diff
@@ -0,0 +1,24 @@
+const db = require("../db/connection");
+
+exports.selectArticleIdComments = async (article_id) => {
+  const checkExists = async (article_id) => {
+    const dbOutput = await db.query(
+      `SELECT * FROM articles WHERE article_id = $1;`,
+      [article_id]
+    );
+    if (dbOutput.rows.length === 0){
+      return Promise.reject({ status: 404, message: "Not Found" });
+    }
+  };
+
+const doesArticleIdExist = await checkExists(article_id);
+  if (doesArticleIdExist) {
+    return doesArticleIdExist;
+  }
```

Comment:
Something to think about:
you don't necessarily need to check if the article exists in ALL cases. In which case do you know you need to do a check like this? (or what kind of result do you need to do this in)